File: twiml/voice/record/record-3/record-3.5.x.cs
using Twilio.TwiML;
using Twilio.Http;
using System;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        response
            .Say("Please leave a message at the beep.\nPress the star key when finished.");
        response.Record(action: new Uri("http://foo.edu/handleRecording.php"),
            method: HttpMethod.Get, maxLength: 20, finishOnKey: "*");
        response.Say("I did not receive a recording");

        Console.WriteLine(response.ToString());;
    }
}

File: twiml/voice/record/record-3/record-3.5.x.js
const VoiceResponse = require('twilio').twiml.VoiceResponse;


const response = new VoiceResponse();
response.say('Please leave a message at the beep.\nPress the star key when finished.');
response.record({
    action: 'http://foo.edu/handleRecording.php',
    method: 'GET',
    maxLength: 20,
    finishOnKey: '*'
});
response.say('I did not receive a recording');

console.log(response.toString());
